assets/src/components/cd/pipelines/nodes/StageNode.tsx

```diff
@@ -131,25 +132,51 @@ const IconHeadingInnerSC = styled.div(({ theme }) => ({
   width: '100%',
 }))
 
+export const PrCountBadgeSC = styled(CountBadge).attrs(() => ({
```

Comment:
we should probably have a design system component that can do this at some point.  Grommet had one they called Stack that effectively let you overlay components at different anchor points which is v useful.